integrations/java/openhouse-java-runtime/src/main/java/com/linkedin/openhouse/javaclient/OpenHouseCatalog.java

```diff
@@ -128,6 +113,37 @@ public void initialize(String name, Map<String, String> properties) {
     this.cluster = properties.getOrDefault(CLUSTER_PROPERTY, DEFAULT_CLUSTER);
   }
 
+  /**
+   * sets the auth token as default header which gets added to every request from ApiClient
+   * @param token
+   */
+  protected void setToken(String token) {
```

Comment:
How about private with VisibilityForTesting? Or maybe move this method to unit-test?

Reply:
setToken needs visibility to subclass which has to update the auth-token in apiClients headers. Hence its protected.